.travis.yml

```diff
@@ -9,6 +9,7 @@ language: php
 matrix:
   fast_finish: true
   include:
+    - { php: 8.1, env: PSR_MESSAGE_BRIDGE_VERSION=2.3 SYMFONY_VERSION=5.4 SYMFONY_DEPRECATIONS_HELPER=weak, name: "PHP 8.1, Symfony 5.4" }
```

Comment:
This matrix should be extended to include Symfony 6.4, as well as all the PHP versions that are compatible with Symfony 5.4 and Symfony 6.4. You'll also need to add a new set of permutations for both Guzzle 6 and Guzzle 7. 

You can remove Symfony version 5.3: it's completely superseded by 5.4.

Reply:
@itafroma Added to the matrix Guzzle 6 and 7 tests, on both PHP 8.1, 8.0 and 7.4. Also, how do I sort out the scrutinizer issue? It looks like it runs with its own `outdated` php_codesniffer which fails. Is it ok if I disable that and install code sniffer on this repo?